docs/troubleshooting/development/i-cant-install-the-headless-cms-plugins-in-my-account.md

```diff
@@ -18,12 +27,56 @@ Installing plugin @vtex/cli-plugin-cms... failed
+```
+</td>
+<td width="30%">
+The VTEX IO CLI’s node version isn’t updated. See that the expected version was `>=14.0.0`, but, in this case, the node version in the VTEX IO CLI was `12.12.0`.
+</td>
+<td width="30%">
+<a href="https://developers.vtex.com/docs/troubleshooting/error-installing-headless-cms-plugins-on-windows#update-vtex-io-cli">Update VTEX IO CLI</a>
+</td>
+</tr>
+</table>
+
+## Solutions
+
+To solve these issues, see the following instructions:
+
+- [Enable Developer Mode on your machine](#enable-developer-mode-on-your-machine)
+- [Update VTEX IO CLI](#update-vtex-io-cli)
+
+### Enable Developer Mode on your machine
+
+FastStore projects rely on creating symbolic links (symlinks), and the [Developer Mode](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development#accessing-settings-for-developers) grants the necessary permissions and privileges to use them, which reduces the probability of encountering errors during development.
+
+> ⚠️ Running the FastStore project as an Administrator isn’t recommended.
+
+Learn how to enable Developer Mode in the Microsoft's official guide [Enable your device for development](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development#accessing-settings-for-developers).
+
+After enabling Developer Mode, launch Windows Terminal and run `vtex cms`. This will allow the system to create the necessary symlinks for running commands from the Headless CMS plugin.
 
-To solve this issue, enable [Developer Mode](https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development#accessing-settings-for-developers) in your machine.
-FastStore projects rely on creating symbolic links (symlinks), and this mode grants the necessary permissions and privileges to use them, which reduces the probability of encountering errors during development.
+### Update VTEX IO CLI
 
-> ⚠️ Running the FastStore project as an Administrator is not recommended.
+Even if you update the `node` version on your machine, you must also keep VTEX IO CLI updated to avoid errors.
 
-1. To enable Developer Mode, refer to Microsoft's official guide [Enable your device for development](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development#accessing-settings-for-developers).
+To update VTEX IO CLI, open the Windows Terminal and follow these instructions:
 
-2. After enabling Developer Mode, launch Windows Terminal and run `vtex cms`. This will allow the system to create the necessary symlinks for running commands from the Headless CMS plugin.
+1. Run the `vtex autoupdate` command in the Windows Terminal.
+2. Run the `vtex -v` command to verify if the VTEX IO CLI and `node` versions are updated.
+3. Run the `vtex plugins install cms` command to install the Headless CMS plugins.
```

Comment:
```suggestion
3. Run the `vtex plugins install cms` command to install the Headless CMS plugins.
4. Check if the installation of Headless CMS plugins was successful by running `vtex cms`. You should see the following:

```sh
$ vtex cms
Syncs CMS schemas with the current workspace in use.

USAGE
     $ vtex cms COMMAND
COMMANDS
     cms sync Syncs CMS schemas with the current workspace in use.
```

```